set.mm

```diff
@@ -123600,8 +123616,16 @@ distinguish between finite and infinite sets (and therefore if the set size
     wf mp2b eqeltri ) ABCCDZEZFGHHDZHBPBITFJKUAHBLBSCCMMNOQR $.
 
   $( Positive integers are a subset of integers.  (Contributed by NM,
-     9-Jan-2002.) $)
+     9-Jan-2002.)  Reduce dependencies on axioms.  (Revised by Steven Nguyen,
+     29-Nov-2022.) $)
   nnssz $p |- NN C_ ZZ $=
+    ( vx cn cz cv wcel cr cc0 wceq cneg w3o nnre 3mix2 elz sylanbrc ssriv ) ABC
+    ADZBEZPFEPGHZQPIBEZJPCEPKQRSLPMNO $.
```

Comment:
In this and related cases, it might be globally shorter to prove the NN0 version and then the NN version from it. Have you had / can you have a look at it in a future PR ?

Reply:
Ok, I see. I guess ideally one would have
```
df-nn0 $a |- NN0 = ( rec ( ( x e. _V |-> ( x + 1 ) ) , 0 ) " om ) $.
df-nn $a |- NN = ( NN0 \ { 0 } ) $.
df-zz $a |- ZZ = { n e. RR | ( n e. NN0 \/ -u n e. NN0 ) } $.
```
but that would be a big piece of work.